CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## v1.4.0
+> Release Date: Unreleased
```

Comment:
```suggestion

> Release Date: Unreleased
```

Reply:
addressed in commit a0e6a231b99ac37596cd1b07cf674e22d71bda11